src/_data/side-nav.yml

```diff
@@ -280,24 +277,21 @@
 - title: Resources
   expanded: false
   children:
-    - title: Overview
-      urlExactMatch: true
-      match-page-url-exactly: true
-      permalink: /resources
-    - title: Books
-      permalink: /resources/books
-    - title: "DartPad in tutorials: Best practices"
-      permalink: /resources/dartpad-best-practices
-    - title: Code of conduct
-      permalink: /code-of-conduct
     - title: FAQ
       permalink: /faq
-    - title: History
+    - title: Language evolution
+      permalink: /guides/language/evolution
+    - title: Language specification
+      permalink: /guides/language/spec    
```

Comment:
**question:** Will these paths change later? This is less a guide than, well, a reference. Will it become `/reference/changelog` or something similar? Same with specification.

Reply:
I _think_ that's the idea. To avoid moving around files too much and having multiple redirects, it's best to leave that decision for later as we finalize our desired directory structure. Trying new locations in the sidenav is cheap :)

Reply:
Yup, what Parker said. I have an issue here to return to it once all the moving is done https://github.com/dart-lang/site-www/issues/4608